api/v1/flow/flow.proto

```diff
@@ -375,6 +375,7 @@ enum DropReason {
     NAT46 = 187;
     NAT64 = 188;
     AUTH_REQUIRED = 189;
+    DSR_ENCAP_GENEVE_NOT_SUPPORT_IPV6_XDP = 190;
```

Comment:
nitpicking: the name looks a bit long to me, how about
```suggestion
    DSR_GENEVE_IPV6_XDP_UNSUPPORTED = 190;
```

Reply:
Thank you for the review! I will make it shorter.